doc/compiler/extensible_classes/Extensible_Classes_in_OMR.md

```diff
@@ -35,8 +35,12 @@ architectures. These are also assigned namespaces as follows:
   - `i386`: for 32-bit x86 (x86-32)
 - `Z`: for the IBM z Systems architecture
 
-Additionally, there is a `TR` namespace, used by all projects. This namespace
-holds all concrete classes.
+In order to provide a concrete name that's usable anywhere, we map all our
```

Comment:
I wonder if it's worth expanding on what we mean by "map" here, given it's not really a straightforward application for the word "map" :) .